### PR TITLE
Fix image resizing to maintain aspect ratio for Wheelright

### PR DESCRIPTION
**Pull request by luke9kim8**
## Fix image resizing to maintain aspect ratio

Issue Number(s): #82 

When we go from desktop to mobile, some images don't resize properly. This PR fixes that.
Since there are so many images to resize, this PR only addresses images in wheelright.

### Checklist

- [x]  Database schema docs have been updated or are not necessary
- [x]  Code follows design and style guidelines
- [x]  Code is commented with doc blocks
- [x]  Tests have been written and executed or are not necessary
- [x]  Latest code has been rebased from base branch (usually `develop`)
- [x]  Commits follow guidelines (concise, squashed, etc)
- [x]  Github issues have been linked in relevant commits
- [x]  Relevant reviewers (Senior Dev/EM/Designers) have been assigned to this PR

### Critical Changes
None

### Related PRs
None

### How to Test
Check all the pages and it should match how it's shown in Figma

### CODE REVIEW

**Comment by luke9kim8** [comment]
I'm not done yet, this is a draft PR

File: src/pages/Location.js [hunk 1]
```diff
@@ -9,27 +9,22 @@ const Location = (props) => {
...
+              </p>
+            </div>
           </ParallaxComponent>
         </div>
       </div>
 
-      <div className="split right">
-        <div className="location-body2" />
-      </div>
+      <div className="right">{/* <div className="location-body2" /> */}</div>
```
Comment (its-edmund) [inline src/pages/Location.js]: nit: looks like this can be removed?

File: src/pages/Statistics.js [hunk 2]
```diff
@@ -4,92 +4,176 @@ import PropTypes from 'prop-types';
...
+          <p className="source">{source}</p>
+        </ParallaxComponent>
       </div>
-    );
-  }
-}
+    </div>
+  );
+};
+// class Statistics extends React.PureComponent {
```
Comment (its-edmund) [inline src/pages/Statistics.js]: nit: Can we also remove this comment as well

**Review by its-edmund — COMMENTED** [review]
LGTM

**Review by its-edmund — APPROVED** [review]

**Pull request merged**